torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -253,6 +253,17 @@ def _is_int8_gemm(inputs):
         def reorder_and_filter(inputs, layout_or_out):
             if _is_int8_gemm(inputs):
                 # No need to reorder for int8 gemm
+                if len(inputs) >= 8:
```

Comment:
Using the inputs len to decide the specific gemm op sounds not general... Possible generalize the mapping with input_indices?

Reply:
Thanks and changed, I think we only need to do the reorder when none default `input_indices`.